bin/flappy_translator.dart

```diff
@@ -1,12 +1,84 @@
+import 'dart:io';
+
+import 'package:yaml/yaml.dart';
+
 import 'package:flappy_translator/flappy_logger.dart';
 import 'package:flappy_translator/flappy_translator.dart';
 
+/// The path to the pubspec file path
+const pubspecFilePath = 'pubspec.yaml';
+
+/// The section id for flappy-translator in the yaml file
+const yamlSectionId = 'flappy_translator';
+
+/// A class of arguments which the user can specify in pubspec.yaml
+class YamlArguments {
+  static const inputFilePath = 'input_file_path';
```

Comment:
May we put the constants in uppercase ?

Reply:
According to the [documentation](https://dart.dev/guides/language/effective-dart/style#prefer-using-lowercamelcase-for-constant-names), constants should be named using lowerCamelCase but I'm happy to change to UPPER_CASE to suit the package's coding style.